<launch>
  <node pkg="haro_description"
        type="pictogram_array_objects_demo.py"
        respawn="true"
        name="pictogram_object_demo_node" />
</launch>